wallet/wallet.sol

```diff
@@ -363,7 +366,7 @@ contract Wallet is multisig, multiowned, daylimit {
     // to determine the body of the transaction from the hash provided.
     function confirm(bytes32 _h) onlymanyowners(_h) returns (bool) {
         if (m_txs[_h].to != 0) {
-            m_txs[_h].to.call.value(m_txs[_h].value)(m_txs[_h].data);
+            require(m_txs[_h].to.call.value(m_txs[_h].value)(m_txs[_h].data));
```

Comment:
Since this now `throws` instead of silently failing, it means that `delete m_txs[_h]` can never be called, and this tx will reside there forever. May not be a problem, not sure yet.